src/screens/Settings/index.tsx

```diff
@@ -108,6 +108,11 @@ function Settings() {
     toggle: toggleDarkTrayIcon,
     setOn: setDarkTrayIcon
   } = useToggle(false)
+  const {
+    on: discordrpc,
+    toggle: toggleDiscordrpc,
+    setOn: setDiscordrpc
+  } = useToggle(false)
```

Comment:
Since you're not getting the current config on the `useEffect` function, the toggle will always change to false when getting back to the settings page.